book/src/artifacts/archives.md

```diff
@@ -0,0 +1,93 @@
+
+
+
+## Auto-Detected Files
+
+We will always auto-detect READMEs, LICENSES, and CHANGELOGs with the following logic (described more below):
+
+* README: [package.readme][config-package-readme], or find `README*`
+* LICENSE: [package.license-file][config-package-license-file], or find `LICENSE*`/`UNLICENSE*`
+* CHANGELOG: find `CHANGELOG*`/`RELEASES*`
+
+"Find `XYZ*`" means we will look for a file whose name starts with "XYZ" in the same directory as the Cargo.toml for a package that defines the app. If no such file is found, we will also search for it in the same directory as the workspace's Cargo.toml (so packages "inherit" these files from the workspace).
+
+It is generally assumed that a directory only contains one of each kind of file. If multiple possible matches are in the same directory we will arbitrarily pick the first one we saw, so don't rely on that.
+
+Auto-detected files are first and foremost [auto-included into the archive](#archive-contents), however they can also be used for other things. For instance, the autodetected CHANGELOG is fed into our CHANGELOG features.
+
+
+
+## Archive Contents
+
+The "root" of an archive is either the actual root directory of the archive (zips); or a directory with the same name as the archive, but without the extension (tarballs). This difference is for compatibility/legacy reasons, and can be smoothed away by unpacking tarballs with tar's `--strip-components=1`.
+
+An app's archive always includes its binaries at the root.
+
+By default [auto-detected files](#auto-detected-files) for a package are auto-included into its archives at the root of the package. The [auto-includes][config-auto-includes] config controls this behaviour.
+
+The [include][config-include] can be used to manually add specific files/directories to the root of the archive.
+
+
+
+## Archive Formats
+
+Archives can be zips or tarballs (gz, xz, or zstd).
+
+By default we make .zip on windows and .tar.xz elsewhere, but this can be configured with [windows-archive][config-windows-archive] and [unix-archive][config-unix-archive] features.
+
+
+
+
+## Build Flags
+
+We currently [always build with `--profile=dist`][dist-profile]
+
+By default we build with `--workspace` [to keep things consistent][workspace-hacks], but this can be configured with the [precise-builds config][config-precise-builds] (see those docs for details on when precise-builds will be force-enabled).
+
+By default we build your packages with default features, but this can be configured with the [features][config-features], [default-features][config-default-features], and [all-features][config-all-features] configs.
+
+When targeting windows-msvc we will unconditionally [append "-Ctarget-feature=+crt-static"][crt-static] to your RUSTFLAGS, which should just be the default for rustc but isn't for legacy reasons.
```

Comment:
A link as to why we think it should be the default would be great here. As a non-Windows dev. I'd love to know more about that.

Reply:
The link on this line is in fact the explanation :)